Blob.js

```diff
@@ -337,7 +337,7 @@
     /*                   Blob constructor                   */
     /********************************************************/
     function Blob (chunks, opts) {
-      chunks = chunks || []
+      chunks = chunks.slice() || []
```

Comment:
if chunks is undefined you'll get an error. You can either use the ternary as I implemented it

```suggestion
      chunks = chunks ? chunks.slice() : []
```

or

```suggestion
      chunks = (chunks || []).slice()
```

Reply:
> if chunks is undefined you'll get an error. You can either use the ternary as I implemented it
> 
> or

Ha ha, my fault